download-windows-esd

```diff
@@ -1,65 +1,65 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
-set -e
+set -Eeuo pipefail -o posix
```

Comment:
Taking the new set options one at a time:

- "-E": not needed. We don't have any subshells here that need to use traps, so no need to introduce a Bash dependency for this.
- "-u": good call.
- "-o pipefail": let's do this too. Per my previous question: I wasn't aware [this was actually POSIX now](https://pubs.opengroup.org/onlinepubs/9799919799/utilities/V3_chap02.html#tag_19_26). Thanks for giving me an impetus to look that up 🙂 
- "-o posix": not needed, because I don't want to make us dependent on Bash in the first place.